Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Use Linux-Alpine image
+FROM amazoncorretto:17-alpine
+
+ARG JAR_LIB_FILE=./app/build/libs/app-all.jar
+# Create directory and switch to it
+WORKDIR /app
+
+# Add project to created folder
```

Comment:
Can we change this comment?  We aren't addint the entire project anymore.

Reply:
Updated